location/models.py

```diff
@@ -44,16 +44,19 @@ def representative_image_or_placeholder(self):
         image = self.representative_image
         return image if image is not None else "/static/img/no_img.png"
 
-    @property
-    def rent_price_for_display(self):
+    def number_of_bed_for_display(self):
         count = self.apartment_set.count()
         if count == 0:
             return None
         elif count == 1:
-            return self.apartment_set.first().rent_price_for_display
+            return self.apartment_set.first().number_of_bed_for_display
         else:
-            minimum = f"{self.apartment_set.order_by('rent_price')[0].rent_price:.0f}"
-            maximum = f"{self.apartment_set.order_by('-rent_price')[0].rent_price:.0f}"
+            minimum = (
+                f"{self.apartment_set.order_by('number_of_bed')[0].number_of_bed:.0f}"
+            )
+            maximum = (
+                f"{self.apartment_set.order_by('-number_of_bed')[0].number_of_bed:.0f}"
+            )
```

Comment:
It appears that we lost some test coverage as well.

![Screen Shot 2019-12-08 at 7 19 58 PM](https://user-images.githubusercontent.com/32490273/70399186-c8810c00-19ef-11ea-9e03-d49ae9bc5898.png)